grammar.js

```diff
@@ -262,6 +265,28 @@ module.exports = grammar({
     keyword_cost: _ => make_keyword("cost"),
     keyword_rows: _ => make_keyword("rows"),
     keyword_support: _ => make_keyword("support"),
+    keyword_extension: _ => make_keyword("extension"),
+    keyword_out: _ => make_keyword("out"),
+    keyword_inout: _ => make_keyword("inout"),
+    keyword_variadic: _ => make_keyword("variadic"),
+
+    keyword_session: _ => make_keyword("session"),
+    keyword_isolation: _ => make_keyword("isolation"),
+    keyword_level: _ => make_keyword("level"),
+    keyword_serializable: _ => make_keyword("seria"),
+    keyword_repeatable: _ => make_keyword("repeatable: _ ="),
+    keyword_read: _ => make_keyword("read: _ =>"),
```

Comment:
Again, I was not able to fin this structure in PG and MariaDB.  Also separating tokens in the keyword statement is not the right way. In SQL there can be multiple spaces/tabs between tokens.

Reply:
Same as before a keyboard macro typo.